packages/elements/src/components/ui/banners/result-list-banner/ResultListBanner.tsx

```diff
@@ -27,11 +27,12 @@ export const ResultListBanner: React.FC<ResultListBannerProps> = ({
       {bannerState.items && (
         <>
           <ResultList>
-            {bannerState.items.map((item) => (
+            {bannerState.items.map((item, index) => (
               <ResultListRow
                 text={item.text}
                 linkText={item.linkText}
                 onClickLink={item.onClickLink}
+                key={`ResultListBannerRow:${index}`}
```

Comment:
Item doesn't have id? Ok!

There's no need for `key` to be globally unique, just `key={index}` is fine! :)

Reply:
oh didn't know that, will change to just index then, thanks!